### ISSUE TITLE
When you use the git flow plugin, it needs to Refresh the repository screen afterward

### ISSUE THREAD

**Issue by cmpalmer66** [gitextensions/gitextensions#7996]
## Current behaviour

When you use the git flow plugin to, say, create a new feature branch, the branch is created and you checked out, but when you dismiss the plugin window or view the git extensions browse window, it doesn't reflect the state of your checkout until you click Repository->Refresh (or hit F5).

## Expected behaviour

Ideally, the browse window should refresh automatically when the checked out repository is changed by git flow.

## Environment

- Git Extensions 3.3.1.7897
- Build 5a97671645532bcedc443bac7b727f40db47cb5c
- Git 2.24.1.windows.2
- Microsoft Windows NT 10.0.18362.0
- .NET Framework 4.8.4075.0
- DPI 96dpi (no scaling)

**Comment by pmiossec** [comment]
You will have to debug it because the code seems good.

When a command is run, a flag is set:
https://github.com/gitextensions/gitextensions/blob/master/Plugins/GitFlow/GitFlowForm.cs#L315

And this value is used to know if the revision grid should be refreshed when the gitflow form is closed :
https://github.com/gitextensions/gitextensions/blob/master/Plugins/GitFlow/GitFlowPlugin.cs#L23

**Comment by pgr1973** [comment]
I agree totally with the issue-opener, after gitflow-operations the left panel does not update itself which is really annoying.

Version-Infos:
- Git Extensions 3.4.3.9999
- Build d4b0f48bbf3e39a71f5d7ff5231be5678d4aa0f1
- Git 2.27.0.windows.1
- Microsoft Windows NT 10.0.18362.0
- .NET Framework 4.8.4220.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Pull requests are welcome.
